Example/Pods/Local Podspecs/PinpointKit.podspec.json

```diff
@@ -6,7 +6,7 @@
     "git": "https://github.com/Lickability/PinpointKit.git",
     "tag": "0.9"
   },
-  "summary": "A library that makes bug reporting dead simple for your testers and users by allowing them to send feedback with annotated screenshots and logs using a simple gesture.",
+  "summary": "A library that makes bug reporting simple for your users by allowing them to send feedback with annotated screenshots and logs.",
```

Comment:
Yeah, we knew about this. Part of the rush up to launch was linting this podspec and shortening this and we launched without copying it back over. nbd